packages/components/src/popover/index.js

```diff
@@ -144,8 +144,10 @@ const Popover = (
 		onFocusOutside,
 		__unstableSlotName = SLOT_NAME,
 		__unstableObserveElement,
-		__unstableForcePosition = false,
+		flip = true,
+		resize = true,
 		__unstableShift = false,
+		__unstableForcePosition = false,
```

Comment:
We should probably remove the default value assignment here, so that the `if ( __unstableForcePosition )` check later in the code can detect any instance in which the `__unstableForcePosition` is passed to `Popover`.

Reply:
👍 I've done that, and update the `if` statement so that it'll show the deprecation message whenever `__unstableForcePosition` is specified as a prop.